modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1036,6 +1036,80 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "Content of form deployed as widget.");
 	}
 
+	@description = "Filter and order the apps created using App Builder"
```

Comment:
Please, move this test to:

https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/app-builder-workflow/app-builder-workflow-test/src/testFunctional/tests/AppBuilderWorkflow.testcase